src/WingetCreateCLI/WingetCreateCLI.csproj

```diff
@@ -77,6 +77,19 @@
       <Generator>PublicResXFileCodeGenerator</Generator>
     </EmbeddedResource>
   </ItemGroup>
+    
+  <!--Include VCLib binary dependencies-->  
+  <ItemGroup Condition="'$(SelfContained)' == true">
```

Comment:
What's selfcontained mean?

Reply:
selfcontained is the flag that indicates whether to publish the .NET runtimes with the exe. I figured we would want the standalone exe to be able to run without any dependencies.

Reply:
Ooh gotcha, that's why the build succeeds when run locally, makes sense